pkg/registry/extensions/podsecuritypolicy/strategy.go

```diff
@@ -54,9 +55,18 @@ func (strategy) AllowUnconditionalUpdate() bool {
 }
 
 func (strategy) PrepareForCreate(ctx genericapirequest.Context, obj runtime.Object) {
+	psp := obj.(*extensions.PodSecurityPolicy)
+	psp.Generation = 1
```

Comment:
why was this added?

Reply:
i copied from Deployment

Reply:
Generation is not related to this PR. Please remove it